appveyor.yml

```diff
@@ -3,6 +3,7 @@
 # Test against these versions of Node.js.
 environment:
   matrix:
+    - nodejs_version: "0.12"
```

Comment:
should've been `git reset --hard`, otherwise you're still on the old working folder state